src/DGmethods/DGmodel.jl

```diff
@@ -82,6 +82,8 @@ function (dg::DGModel)(dQdt, Q, ::Nothing, t; increment=false)
     communicate && MPIStateArrays.start_ghost_exchange!(Qvisc)
   end
 
+  update_aux_diffusive!(dg, bl, Q, t)
```

Comment:
If you have changed auxstate you'll need to either call update aux again after the viscous state has been communicated or recommunicated the aux state.

Of course you will also probably want some catch for whether aux has changed so you know whether the comm is needed

Reply:
hmmm... this function updates the auxstate and relies on knowing the full diffstate. 

Should I put a call to `MPIStateArrays.finish_ghost_recv!(Qvisc)` before L85? Do I need to add another `MPIStateArrays.start_ghost_exchange!(auxstate)` afterwards? I'm not sure what the exact logic is for these.